include/NovelRT/Maths/GeoVector2F.h

```diff
@@ -40,23 +40,46 @@ namespace NovelRT::Maths
         float x;
         float y;
 
+        /**
+         * @brief Construct new GeoVector2F with X and Y components set to 0
+         * @returns Newly constructed GeoVector2F
+         */
         GeoVector2F() noexcept : x(0.0f), y(0.0f)
         {
         }
+
+        /**
+         * @brief Construct new GeoVector2F with X and Y components with x and y paramters
+         * @param x Set X Component Value
+         * @param y Set Y Component Value
+         * @returns Newly constructed GeoVector2F
+         */
         GeoVector2F(float x, float y) noexcept : x(x), y(y)
         {
         }
 
+        /**
+         * @brief Compute a normalized vector of this vector
+         * @returns A Unit Vector in GeoVector2F that have the length of 1
+         */
         inline GeoVector2F getNormalised() const noexcept
         {
             return GeoVector2F(glm::normalize(*reinterpret_cast<const glm::vec2*>(this)));
         }
 
+        /**
+         * @brief Compute a magnitude from 0, 0 Coordinate Vector
+         * @returns Magnitude measured from 0, 0 Coordinate Vector
+         */
         inline float getMagnitude() const noexcept
         {
             return glm::length(*reinterpret_cast<const glm::vec2*>(this));
         }
 
+        /**
+         * @brief Compute length
+         * @returns Vector Length, also known as Magnitude
+         */
```

Comment:
Made it more consistent with getMagnitude, did some cleanup.
```suggestion
        /**
         * @brief Computes the length of this GeoVector2F instance.
         * @returns Length represented as a float. This is the same as the magnitude.
         */
```